### DISCUSSION TITLE
Email settings have been made but it is not testing or sending. (Mail could not be sent. No additional error message provided. Check your mail settings and your app log.)

### DISCUSSION THREAD

**Discussion by mdedeoglu2** [snipe/snipe-it#14367] [Q&A]
Snipe-IT version:v6.3.1 build 12672 (g00cea3eb3)
PHP Version:8.1.2-1ubuntu2.14
Timezone:Europe/Istanbul
Mail From Address:IT <mdedeoglu@koyuncuoglu.com.tr>
Table Storage:cookieStorage
Software License:[AGPL3](https://www.gnu.org/licenses/agpl-3.0.en.html)
Laravel Version:8.83.27
Database Driver:mysql
Mail Reply-To Address:IT <mdedeoglu@koyuncuoglu.com.tr>
![env mail ayar](https://github.com/snipe/snipe-it/assets/162312737/d317e212-64ff-40cb-b297-fd0258cf8edc)

**I tried Snipe It with a fresh installation on Centos and Ubuntu, but it does not send e-mails.
- There is no relevant data in /var/www/html/storage/logs/laravel.log.
- What other logs can I check in the Ubuntu system?
![mail error message](https://github.com/snipe/snipe-it/assets/162312737/0ae14846-d6ff-47ac-b477-18f36e39d9fd)

- I would be happy if you could support me with what can be done for a solution.**

**Comment by mdedeoglu2** [comment]
**We cannot use Gmail because 'less secure app' support has been removed.**

**Comment by ksgrg** [comment]
I have got the following settings configured. We use Office 365 and it seems to be sending emails without any issues.
Check if your mail server has TLS and port 587 configured.

Do a telnet from your server to your mail server port 587 and see if the port is open

![image](https://github.com/snipe/snipe-it/assets/162708626/e7beaa43-2c7b-409c-b940-05df1a39bd65)

MAIL_DRIVER=smtp
MAIL_HOST=smtp.office365.com
MAIL_PORT=587
MAIL_USERNAME=asset@domain.com
MAIL_PASSWORD=password
MAIL_ENCRYPTION=tls
MAIL_FROM_ADDR=asset@domain.com
MAIL_FROM_NAME='Asset Management'
MAIL_REPLYTO_ADDR=asset@domain.com
MAIL_REPLYTO_NAME='Asset Management'
MAIL_AUTO_EMBED_METHOD='attachment'

The above is what I have configured and it works beautifully.
